### ISSUE TITLE
Be able to estimate job build time

### ISSUE THREAD

**Issue by expipiplus1** [hercules-ci/hercules#44]
It would be super to be able to estimate job build time. This might enable us to do some clever load balancing.

http://www.cs.uccs.edu/~xzhou/teaching/CS551/Projects/HeavyTailedTaskAssignment_JACM02.pdf

**Comment by domenkozar** [comment]
That's really hard to do if you share resources between builds (parallel building).

**Comment by expipiplus1** [comment]
yeah, perhaps there would have to be some measure of how loaded the machine is, or we could measure cpu time.

**Comment by domenkozar** [comment]
We could monitor the chroot resources for a build and based on that usage estimate it's share. But that sounds like voodoo :)
